src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -49,17 +49,26 @@ using raylet_scheduling_policy::SchedulingResultStatus;
 
 using ScheduleMap = absl::flat_hash_map<BundleID, NodeID, pair_hash>;
 
+struct SchedulePgRequest {
+  std::shared_ptr<GcsPlacementGroup> placement_group;
+  PGSchedulingFailureCallback failure_callback;
+  PGSchedulingSuccessfulCallback success_callback;
+};
+
+struct SchedulePreparedPgRequest {
+  SchedulePgRequest request;
+  std::vector<std::shared_ptr<const BundleSpecification>> prepared_bundles;
```

Comment:
Seems we don't need `SchedulePreparedPgRequest` as we can get bundles from `SchedulePgRequest.placement_group`